docs/API.md

```
@@ -130,6 +143,14 @@ Deletes a hospital.
 **Response**
 - Status Code: 204 No Content
 
+### API Documentation
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)